### ISSUE TITLE
Empty list items

### ISSUE THREAD

**Issue by acurrieclark** [tzurbaev/laravel-forge-api#16]
In my use case there is the occasional scenario where i request a list of resources from a server where none of that particular resource exist. For example asking for a list of databases on the server.

At the moment, because the result is an empty array, an `InvalidArgumentException` is thrown. I wonder if the [current check for valid response](https://github.com/tzurbaev/laravel-forge-api/blob/master/src/Commands/ResourceCommand.php#L142) might be better changed from:

```php
<?php
if (empty($json[$itemsKey])) {
    throw new InvalidArgumentException('Given response is not a '.$this->resourcePath().' response.');
}
```
to
```php
<?php
if (!isset($json[$itemsKey])) {
    throw new InvalidArgumentException('Given response is not a '.$this->resourcePath().' response.');
}
```

This would allow an empty array to be returned if indeed there were no items, but would still throw an exception if the wrong response was returned.

**Comment by tzurbaev** [comment]
Thank you for reporting! This should be fixed in `1.4.1`. Please, run `composer update tzurbaev/laravel-forge-api` and check if the issue is resolved now.

**Comment by acurrieclark** [comment]
Welcome as always.
